realtime/connection.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, url: str, auto_reconnect: bool = False, params: Dict[str, Any
         self.kept_alive = False
         self.auto_reconnect = auto_reconnect
 
-        self.channels = cast(defaultdict[str, List[Channel]], self.channels)
+        self.channels = cast(DefaultDict[str, List[Channel]], self.channels)
```

Comment:
Actually I'm unsure why we do a cast like this in the first place. What do you think about editing L44 to
```py
self.channels: DefaultDict[str, List[Channel]] = defaultdict(list)
``` and removing this line with the cast?

Reply:
That's an even better approach. +1

Reply:
@anand2312 - I've updated the merge request as per your suggestion.